jest.config.ts

```diff
@@ -38,38 +38,40 @@ const config: Config = {
         "^react$": "<rootDir>/node_modules/react",
         "^react-dom$": "<rootDir>/node_modules/react-dom",
         "^matrix-js-sdk$": "<rootDir>/node_modules/matrix-js-sdk/src",
-        "^matrix-react-sdk$": "<rootDir>/node_modules/matrix-react-sdk/src",
-        "decoderWorker\\.min\\.js": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
-        "decoderWorker\\.min\\.wasm": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
-        "waveWorker\\.min\\.js": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
-        "context-filter-polyfill": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
-        "FontManager.ts": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/FontManager.js",
-        "workers/(.+)Factory": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/workerFactoryMock.js",
+        "^matrix-react-sdk$": "<rootDir>/linked-dependencies/matrix-react-sdk/src",
+        "decoderWorker\\.min\\.js": "<rootDir>/linked-dependencies/matrix-react-sdk/__mocks__/empty.js",
+        "decoderWorker\\.min\\.wasm": "<rootDir>/linked-dependencies/matrix-react-sdk/__mocks__/empty.js",
+        "waveWorker\\.min\\.js": "<rootDir>/linked-dependencies/matrix-react-sdk/__mocks__/empty.js",
+        "context-filter-polyfill": "<rootDir>/linked-dependencies/matrix-react-sdk/__mocks__/empty.js",
+        "FontManager.ts": "<rootDir>/linked-dependencies/matrix-react-sdk/__mocks__/FontManager.js",
+        "workers/(.+)Factory": "<rootDir>/linked-dependencies/matrix-react-sdk/__mocks__/workerFactoryMock.js",
         "^!!raw-loader!.*": "jest-raw-loader",
-        "recorderWorkletFactory": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
+        "recorderWorkletFactory": "<rootDir>/linked-dependencies/matrix-react-sdk/__mocks__/empty.js",
         "^fetch-mock$": "<rootDir>/node_modules/fetch-mock",
         // :TCHAP:
         "MImageBody": "<rootDir>/src/tchap/customisations/components/views/messages/ContentScanningImageBody.tsx",
         "../../../../../../src/tchap/components/views/messages/OriginalFileBody":
-            "<rootDir>/node_modules/matrix-react-sdk/src/components/views/messages/MImageBody.tsx",
+            "<rootDir>/linked-dependencies/matrix-react-sdk/src/components/views/messages/MImageBody.tsx",
         "MAudioBody": "<rootDir>/src/tchap/customisations/components/views/messages/ContentScanningAudioBody.tsx",
         "../../../../../../src/tchap/components/views/messages/OriginalAudioBody":
-            "<rootDir>/node_modules/matrix-react-sdk/src/components/views/messages/MAudioBody.tsx",
+            "<rootDir>/linked-dependencies/matrix-react-sdk/src/components/views/messages/MAudioBody.tsx",
         "MStickerBody": "<rootDir>/src/tchap/customisations/components/views/messages/ContentScanningStickerBody.tsx",
         "~tchap-web-dep/(.*)": "<rootDir>/yarn-linked-dependencies/$1",
```

Comment:
can we delete the shorcut to yarn-linked-dependencies ? (or comment it out, if it feels too risky)